Library/TimerLabel.swift

```diff
@@ -1,65 +1,51 @@
 import UIKit
 
 public final class TimerLabel: UILabel {
+    public typealias Seconds = Int
     private let countingInterval: Double = 1
     private var _timer: Timer?
     public var isPaused = false
     public var isRunning = false
     private var referenceDate = Date()
-    private var counter: TimeInterval = 0 {
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/7d4b7a47f520e10c677d21891d119ba0b38f461e/Library/TimerLabel.swift#L10"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>TimerLabel.swift:10</code></td>
    </tr>
  </tbody>
</table>